tooling/metamodel-generator/src/main/java/org/hibernate/jpamodelgen/annotation/CriteriaFinderMethod.java

```diff
@@ -69,9 +69,13 @@ public String getAttributeDeclarationString() {
 						.append(");");
 			}
 		}
+		String localSessionName = isReactiveSession() ? "_session" : sessionName;
```

Comment:
Not sure how keen I am on `_session`. It's not the worst thing on Earth, but up until this point I managed to keep the generated code looking quite human-written. Is the `_` really necessary here?

Reply:
No, I just want it to be a different name than the session `Uni`, I can find better.